src/ansys/dpf/core/data_sources.py

```diff
@@ -18,7 +18,8 @@
 )
 
 from ansys.dpf.core.check_version import version_requires
-
+from ansys.grpc.dpf import data_sources_pb2
```

Comment:
@MichaelNale sadly if I remember correctly we on purpose do not directly import from `ansys.grpc.dpf`. However, the integration of it in `ansys-dpf-core`, I think this may not be an issue.